temporal-sdk/src/main/java/io/temporal/common/interceptors/WorkflowInboundCallsInterceptor.java

```diff
@@ -128,4 +130,22 @@ public Object getResult() {
 
   /** Called when a workflow is queried. */
   QueryOutput handleQuery(QueryInput input);
+
+  /**
+   * Intercepts creation of the workflow main method thread
+   *
+   * @param runnable thread function to run
+   * @param name name of the thread, optional
+   * @return created WorkflowThread
```

Comment:
Change comment to smth like:

@return created workflow thread. Treat the result as an opaque object that shouldn't be manipulated in any way by the interceptor code.